src/components/routes/Host/ListingForm/ListingFormNav/ListingFormNav.container.ts

```diff
@@ -1,72 +1,38 @@
 import styled from 'styled-components';
-import { color, cover, typography } from 'styled/utils';
+import { color, typography } from 'styled/utils';
 
-const ListingFormNavContainer = styled.div`
+const ListingFormNavMobileContainer = styled.div`
   background-color: ${color('lighter')};
-  height: 64px;
-
+  height: 90px;
+  width: 100%;
 
   .bee-general-wrapper {
     display: flex;
+    flex-direction: column;
     justify-content: space-between;
     height: 100%;
     margin: 0 auto;
-
+    min-width: auto;
```

Comment:
same here

Reply:
removed width="100%" as props passed to GeneralWrapper and declared `width` and `min-width` here as well